CHANGELOG.md

```diff
@@ -202,10 +212,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Packages
     - upgrade to `ember-animated@0.8.1`
 
-### Removed
-- Tests
-    - `taxonomy` FactoryGuy factory
-
```

Comment:
But this *was* removed in 19.10.0: https://github.com/CenterForOpenScience/ember-osf-web/compare/19.9.0...19.10.0#diff-d597d2172d52d886f5b0785fe92e035c

Reply:
Ah, right right! 🤕

Reply:
Oh sorry, I force pushed to fix the tests and ended up wiping out your initial changelog fix.

Reply:
no worries, man!